channels/tr.m3u

```diff
@@ -15,6 +15,10 @@ https://live.al-zahratv.com/live/playlist2/index.m3u8
 http://stream.taksimbilisim.com:1935/altastv/bant1/playlist.m3u8
 #EXTINF:-1 tvg-id="ARASTV.tr" tvg-name="ARAS TV" tvg-country="TR" tvg-language="" tvg-logo="" group-title="",ARAS TV
 http://1.rtmp.org/tv217/yayin.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="TR" tvg-language="Arabic" tvg-logo="" group-title="",Belqees TV
+https://svs.itworkscdn.net/itwlive/itw3.smil/playlist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="TR" tvg-language="Arabic" tvg-logo="" group-title="",Belqees TV
+https://query-streamlink.lanesh4d0w.repl.co/iptv-query?streaming-ip=https://www.youtube.com/c/BelqeesTV/live
```

Comment:
Belqees TV is a Yemeni channel according to this: https://www.lyngsat.com/tvchannels/ye/Belqees-TV.html

Reply:
Thanks fixed